chef-dk-gemset/chef-dk-gemset.gemspec

```diff
@@ -0,0 +1,77 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'chef-dk-gemset/version'
```

Comment:
can we make this just chef-dk/version and nuke the whole lib structure under here?